builder/azure/arm/builder.go

```diff
@@ -51,11 +51,11 @@ func (b *Builder) Prepare(raws ...interface{}) ([]string, error) {
 	return warnings, errs
 }
 
-func (b *Builder) Run(ui packer.Ui, hook packer.Hook) (packer.Artifact, error) {
+func (b *Builder) Run(ctx context.Context, ui packer.Ui, hook packer.Hook) (packer.Artifact, error) {
 
 	ui.Say("Running builder ...")
 
-	ctx, cancel := context.WithCancel(context.Background())
+	ctx, cancel := context.WithCancel(ctx)
 	b.ctxCancel = cancel
```

Comment:
this property can now be removed from the builder, since we don't need to propagate cancellation in the builder anymore.

Reply:
oh right, nice one ! Removing it.